ops.h

```diff
@@ -191,15 +238,15 @@ HWY_INLINE void FullDotProductsForStrip(
 
 }  // namespace detail
 
-// Stores dot products of rows with `vec_aligned` to a buffer, then stores them
-// to `out`.
-template <size_t kOuter, size_t kInner, typename MatT, size_t kCapacity,
-          typename VecT>
-HWY_INLINE void MatVec(const CompressedArray<MatT, kCapacity>& mat,
-                       const size_t mat_ofs,
-                       const VecT* HWY_RESTRICT const vec_aligned,
-                       float* HWY_RESTRICT out, hwy::ThreadPool& pool) {
-  PROFILER_ZONE("MatVec");
+// Stores dot products of rows with `vec_aligned` + add to a buffer (if kAdd),
```

Comment:
If we have other changes, consider changing 'add to' to 'add the values from `add`'.